src/main/java/teammates/ui/controller/InstructorCourseStudentDetailsEditSaveAction.java

```diff
@@ -57,14 +57,24 @@ public ActionResult execute() throws EntityDoesNotExistException {
             
             boolean isSectionChanged = student.isSectionChanged(originalStudentAttribute);
             boolean isTeamChanged = student.isTeamChanged(originalStudentAttribute);
+            boolean isEmailChanged = student.isEmailChanged(originalStudentAttribute);
             if (isSectionChanged) {
                 logic.validateSectionsAndTeams(Arrays.asList(student), courseId);
             } else if (isTeamChanged) {
                 logic.validateTeams(Arrays.asList(student), courseId);
             }
             
             logic.updateStudent(studentEmail, student);
-            statusToUser.add(new StatusMessage(Const.StatusMessages.STUDENT_EDITED, StatusMessageColor.SUCCESS));
+            
+            boolean isSessionSummarySendEmail = getRequestParamAsBoolean(Const.ParamsNames.SESSION_SUMMARY_EMAIL_SEND_CHECK);
+            if (isEmailChanged && isSessionSummarySendEmail) {
+                logic.sendFeedbackSessionsSummaryOfCourseToNewStudentEmail(courseId, student);
```

Comment:
Instead of having to connect to logic, simply write the functions to generate and send the email here:
```java
try {
    EmailWrapper email = new EmailGenerator().generateFeedbackSessionSummaryOfCourse(courseId, student);
    new EmailSender().sendEmail(email);
} catch (Exception e) {
    log.severe(...); // write an appropriate logging message
}
```
This way there is no need for `Logic` and `FeedbackSessionsLogic` to be bloated with email-sending functions.